src/popup/components/AccountCardTotalTokens.vue

```diff
@@ -8,6 +8,7 @@
       <span class="digit">
         {{ totalTokens }}
       </span>
+      {{ }}
```

Comment:
?

Reply:
Adds a space between the number of tokens and the word "tokens"
`0Tokens => 0 Tokens`